src/view/ChewingEditor.cpp

```diff
@@ -178,6 +178,27 @@ void ChewingEditor::showAbout()
     aboutBox.exec();
 }
 
+void ChewingEditor::showDeleteConfirmWindow()
+{
+    QString text = QString("Do you want to delete this phrase?");
+
+    QMessageBox deleteBox(this);
```

Comment:
This is common in Windows, and I don't see many in Linux.
But in my opinion, add Icon is OK.